File: src/main/Java/boj_step10/boj3009.java
import java.io.*;
import java.util.StringTokenizer;

public class boj3009 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int[] xList = new int[3];
        int[] yList = new int[3];
        for (int i = 0; i < 3; i++) {
            StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
            xList[i] = Integer.parseInt(tokenizer.nextToken());
            yList[i] = Integer.parseInt(tokenizer.nextToken());
        }
        int x, y;
        if (xList[0] == xList[1]) x = xList[2];
        else if (xList[0] == xList[2]) x = xList[1];
        else x = xList[0];
        if (yList[0] == yList[1]) y = yList[2];
        else if (yList[0] == yList[2]) y = yList[1];
        else y = yList[0];
        writer.append(x + " " + y).close();
    }
}

File: src/main/boj_step10/boj3009.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val xList = mutableListOf<Int>()
    val yList = mutableListOf<Int>()
    repeat(3) {
        val (x, y) = r.readLine().split(" ").map { it.toInt() }
        xList.add(x)
        yList.add(y)
    }
    val x = if(xList[0] == xList[1]) xList[2] else if (xList[0] == xList[2]) xList[1] else xList[0]
    val y = if(yList[0] == yList[1]) yList[2] else if (yList[0] == yList[2]) yList[1] else yList[0]
    w.write("$x $y")
    w.close()
}